File: javax/src/test/java/me/panpf/javax/test/lang/ThrowableTest.java
package me.panpf.javax.test.lang;

import me.panpf.javax.lang.Throwablex;
import org.junit.Assert;
import org.junit.Test;

public class ThrowableTest {

    @Test
    public void test() {
        String result = Throwablex.stackTraceToString(new Throwable());
        Assert.assertNotNull("stackTraceToString", result);
    }
}

File: javax-kt/src/test/java/me/panpf/javaxkt/test/lang/ThrowableTest.kt
package me.panpf.javaxkt.test.lang

import me.panpf.javaxkt.lang.stackTraceToString
import org.junit.Assert
import org.junit.Test

class ThrowableTest {

    @Test
    fun test() {
        val result = Throwable().stackTraceToString()
        Assert.assertNotNull("stackTraceToString", result)
    }
}
